website/docs/r/online_archive.html.markdown

```diff
@@ -113,6 +114,10 @@ The only field required for criteria type `CUSTOM`
 
 * `expire_after_days` - Number of days used in the date criteria for nominating documents for deletion. Value must be between 7 and 9215.
 
+### Data Process Region
+* `cloud_provider` - Human-readable label that identifies the Cloud service provider where you wish to store your archived data.
+* `region` - Human-readable label that identifies the geographic location of the region where you wish to store your archived data. (`US_EAST_1`, `US_WEST_2`, `SA_EAST_1`, `EU_WEST_1`, `EU_WEST_2`, `EU_CENTRAL_1`, `AP_SOUTH_1`, `AP_SOUTHEAST_2`)
```

Comment:
nit: instead of hardcoding valid values for region we can reference the API docs.

Reply:
let me know if new changes are ok

Reply:
sorry for not stating beforehand, it is the same case for data sources so we keep docs aligned

Reply:
sorry, it was my bad - thanks for catching that!